projects/plugins/jetpack/extensions/blocks/button/button.php

```diff
@@ -211,6 +221,34 @@ function get_button_styles( $attributes ) {
 		$styles[] = 'max-width: 100%';
 	}
 
+	if ( $has_custom_border_color ) {
+		$border_styles['color'] = $attributes['style']['border']['color'];
+	}
+
+	if ( $has_border_style ) {
+		$border_styles['style'] = $attributes['style']['border']['style'];
+	}
+
+	if ( $has_border_width ) {
+		$border_styles['width'] = $attributes['style']['border']['width'];
+	}
+
+	if ( $has_individual_borders ) {
+		foreach ( array( 'top', 'right', 'bottom', 'left' ) as $side ) {
+			$border                 = $attributes['style']['border'][ $side ] ?? null;
+			$border_side_values     = array(
+				'width' => $border['width'] ?? null,
+				'color' => $border['color'] ?? null,
+				'style' => $border['style'] ?? null,
+			);
+			$border_styles[ $side ] = $border_side_values;
+		}
+	}
+
+	if ( isset( wp_style_engine_get_styles( array( 'border' => $border_styles ) )['css'] ) ) {
+		$styles[] = wp_style_engine_get_styles( array( 'border' => $border_styles ) )['css'];
+	}
```

Comment:
Could call `wp_style_engine_get_styles` once here:

```suggestion
	$border_styles = wp_style_engine_get_styles( array( 'border' => $border_styles ) );
	if ( isset( $border_styles['css'] ) ) {
		$styles[] = $border_styles['css'];
	}
```

Reply:
Oh well spotted!